bmaptools/CLI.py

```diff
@@ -642,6 +645,10 @@ def parse_arguments():
     text = "do not verify the data checksum while writing"
     parser_copy.add_argument("--no-verify", action="store_true", help=text)
 
+    # The --psplash-pipe option
```

Comment:
A couple of points.

1. Surely this option should get the pipe file path? I do not see it. Instead, it is a true/false option.  args.psplash_pipe will be one of True/False. And then the code above will try to open a file of type boolean. Is this intended?

2. I suggest to name the option in a general manner. Psplash is a Yocto-specific thing. For example:
--progress-pipe.

Reply:
1. It should be a string. I'm not sure why are you pointing to a boolean.
2. That's the problem. What we write to the pipe is process specific. Psplash expects "PROGRESS X" and this flag adheres to that interface.

Reply:
@dedekind are you ok with closing this? I'm not sure if there is something I can improve here.